File: gestion_personnel_front/src/app/models/pwdForm/pwd-form.ts
export class PwdForm {

    tokenId: number;
    password: string;
    confirmedPassword: string;

    constructor(
        tokenId: number,
        password: string,
        confirmedPassword: string
    ) {
        this.tokenId = tokenId;
        this.password = password;
        this.confirmedPassword = confirmedPassword;
    }
}

File: gestion_personnel/src/main/java/com/cdaprojet/gestion_personnel/model/pwdForm/PwdForm.java
package com.cdaprojet.gestion_personnel.model.pwdForm;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PwdForm {
    
    private long tokenId;
    private String password;
    private String confirmedPassword;

}
